modules/login/test/Reset_Password_Test_Plan.md

```diff
@@ -1,12 +1,16 @@
 # Reset Password Test Plan
```

Comment:
@johnsaigle should a mention of the usePwnedPasswordsAPI configuration option be present in this test plan and the readme for that module ?

doesn't it affect the choice of password ?

Reply:
Good point, I'll add it.